specification/sigma-correlation-rules-specification.md

```diff
@@ -465,6 +500,141 @@ correlation:
 Note:
 Even if the rule many_failed_logins groups by the "ComputerName" field, the correlation rule only uses its own `group-by` "User".
 
+#### Value Sum (value_sum)
+
+Check if the `sum` of a number field match a limit
+
+Requires:
+
+- `group-by`
+- `timespan`
+- `condition`
+- `field` in condition section.
+
+```yaml
+title: Possible Exfiltration to Website
+correlation:
+    type: value_sum
+    rules:
+        - website_access
+    group-by:
+        - SourceIP
+        - User
+    timespan: 1h
+    condition:
+        field: bytes_sent
+        gt: 1000000
+```
+
+#### Value Max (value_max)
```

Comment:
Asking myself if this makes sense as correlation rule. I can also match on values greater than on individual events and get the same result, except that multiple detections are possibly aggregated, which can or should be done by a SOAR or case management anyways.